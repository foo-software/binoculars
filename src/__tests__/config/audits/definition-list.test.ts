import auditDefinitionList from '../../../config/audits/definition-list';

describe('definition-list audit', () => {
  it('should match snapshot', () => {
    expect(auditDefinitionList('en')).toMatchSnapshot();
  });
});
