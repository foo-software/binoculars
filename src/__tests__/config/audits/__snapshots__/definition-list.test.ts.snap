// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`definition-list audit should match snapshot 1`] = `
Object {
  "description": "When definition lists are not properly marked up, screen readers may produce confusing or inaccurate output. [Learn more](https://web.dev/definition-list/).",
  "title": "\`<dl>\`'s contain only properly-ordered \`<dt>\` and \`<dd>\` groups, \`<script>\`, \`<template>\` or \`<div>\` elements.",
}
`;
